src/components/Registration/RegistrationContainer.tsx

```diff
@@ -35,6 +36,13 @@ export const RegistrationContainer = withRouter(props => {
    */
   const customWindow = window as ICustomWindow;
 
+  const urlDomainPort =
+    customWindow._env_.IO_ONBOARDING_PA_API_HOST +
```

Comment:
since this looks like required environment variables, actually there will be no errors in case these are undefined here and the application will fail with some obscure console message. I suggest - in another PR - to wrap the getter of these vars in order to show a meaningful error message (when not found) and possibly fail on startup, see:

https://github.com/teamdigitale/io-messages-web/blob/master/src/utils/config.ts#L10

moreover that approach (using an interface) will protect against typos.

Reply:
added story https://www.pivotaltracker.com/story/show/169421448 to keep track
about the interface, is it the one i defined in ICustomWindow (that is equivalent to IConfig in the link you provided)?

Reply:
yes